pallet/account-migration/src/benchmarking.rs

```diff
@@ -130,15 +129,7 @@ mod benchmarks {
 					<T as darwinia_staking::Config>::MaxUnstakings::get()
 						as usize
 				]),
-				unstaking_kton: BoundedVec::truncate_from(vec![
```

Comment:
What will happen to unstaking kton and deposits in account migration storage?

Reply:
Those unclaimed accounts with KTON asset in the old ledger will migrated to the new ledger structure, kton either staking or not  will be released while the user sends migrate transaction.